lucet-wasi/README.md

```diff
@@ -2,78 +2,51 @@
 
 Experimental WASI embedding for the Lucet runtime.
 
-Much of this code is a direct port of the `cloudabi-utils`-based syscall emulation layer via
-[`wasmtime`](https://github.com/CraneStation/wasmtime/tree/master/wasmtime-wasi/sandboxed-system-primitives). It
-is currently suitable for demonstration purposes, but needs to be rewritten in a more Rust-native
-style to reduce the code complexity and the number of potential panics.
-
-If you have questions or suggestions, the authors of `lucet-wasi` and others in the WASI community
-can be found in [`#wasi` on Mozilla IRC](https://wiki.mozilla.org/IRC).
+If you have questions or suggestions, the authors of `lucet-wasi` and others
+in the WASI community can be found in the [WebAssembly Discord
+room](https://discord.gg/PtuCxYc).
```

Comment:
My understanding was that the WebAssembly community decided against using that discord, and are selecting an alternative.

Reply:
Ok, so maybe we should just remove this until the new place to talk about WASM and WASI has been figured out :)